packages/itwinui-react/src/core/LabeledSelect/LabeledSelect.types-test.tsx

```diff
@@ -66,3 +66,41 @@ import LabeledSelect from './LabeledSelect.js';
     </>
   );
 };
+
+() => {
+  return (
+    <>
+      <LabeledSelect<string>
+        label='Select Label'
+        options={[
+          // There should be error: TS 2322
+          // @ts-expect-error (TS 2322): Type 'number' is not assignable to type 'string'.
+          { value: 1, label: 'Item #1' },
+        ]}
+        onChange={(value) => {
+          const returnValue: string = value;
+          return returnValue;
+        }}
+      />
+    </>
+  );
+};
+
+() => {
+  return (
+    <>
+      <LabeledSelect<string>
+        label='Select Label'
+        options={[
+          // There should be error: TS 2322
+          // @ts-expect-error (TS 2322): Type 'number' is not assignable to type 'string'.
+          { value: 1, label: 'Item #1' },
+        ]}
+        onChange={(value) => {
+          const returnValue: string = value;
+          return returnValue;
+        }}
+      />
+    </>
+  );
+};
```

Comment:
Is this accidentally the duplicate of the above test?

Reply:
I mean these [two tests](https://github.com/iTwin/iTwinUI/commit/9c481a7eba447414d1133d44195fffe78f3a88a3#diff-e1f949a6c475a77c78a59a84858d06b14cb56c02d3cd0e358f6edf5e39cc7a4dR70-R106) seem to have identical code.